src/ansys/geometry/core/designer/design.py

```diff
@@ -1220,3 +1220,139 @@ def _update_design_inplace(self) -> None:
 
         # Read the existing design
         self.__read_existing_design()
+
+    def update_from_tracker(self, tracker_response):
```

Comment:
I would name this method as protected (i.e. ``_update_from_tracker``). Ideally users shouldn't have to call it

Reply:
makes sense, done.